src/applications/personalization/account/manifest.json

```diff
@@ -0,0 +1,8 @@
+{
+  "appName": "Account",
+  "entryFile": "./account-entry.jsx",
+  "entryName": "account",
+  "rootUrl": "/account",
+  "production": true,
+  "contentPage": "account/index.md"
```

Comment:
This has to be allowed into prod because it shared the /account route with the legacy profile.